src/js/views/patients/patient/data-events/data-events_views.js

```diff
@@ -166,13 +166,38 @@ const FlowItemView = View.extend({
   },
 });
 
+const PatientEventItemView = View.extend({
+  className: 'table-list__item',
+  tagName: 'tr',
+  behaviors: [RowBehavior],
+  template: hbs`
+    <td class="table-list__cell">
+      <span class="patient__event-icon">{{far "paper-plane"}}</span>{{~ remove_whitespace ~}}
+      <span class="patient__action-name">{{ @intl.patients.patient.dataEvents.dataEventsViews.patientEventItemView.checkInCompleted }}</span>
```

Comment:
`patient__action-name` likely needs to be renamed to something about how it is functionally and not "action"